code/modules/power/supermatter/supermatter.dm

```diff
@@ -178,7 +195,14 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	var/bullet_energy = 2
 	///How much hallucination should we produce per unit of power?
 	var/hallucination_power = 0.1
-
+	///The efficiency of the antinoblium interaction.
+	var/antinoblium_efficiency = 0.0
+	///Delta from antinoblium_efficiency and antinoblium composition.
+	var/antinoblium_efficiency_delta = 0.0
```

Comment:
Why does this var need to be stored on the sm?

Reply:
Because if it approaches to 100%, it will have a multiplier of infinite or undefined. If it's just close to 100%, then its effects are extreme, so it should be hard to reach that level, and the delta will prevent memes like "Put a bunch of antinob in SM for NaN power".

Reply:
Oh right I could just remove tbat then apply it to antinob_efficiency